src/components/CautionaryContact/CautionaryContactTable.test.js

```diff
@@ -1,16 +1,25 @@
 import { render } from '@testing-library/react'
 import UserContext from '../UserContext'
-import CautionaryContactView from './CautionaryContactView'
+import CautionaryContactTable from './CautionaryContactTable'
 import { operative } from 'factories/operative'
-describe('CautionaryContactView component', () => {
-  const query = {
-    cautContact: ['VA', 'DAT', 'CIT'],
+import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlertsDescription'
+
+describe('CautionaryContactTable component', () => {
+  const props = {
+    cautionaryContacts: CAUTIONARY_CONTACT,
+    query: {
+      cautContact: ['VA', 'DAT', 'CIT'],
+    },
```

Comment:
this should be `query: ['VA', 'DAT', 'CIT']` as we are passing `searchQuery.query.cautContactCodes` on the `CautionaryContactView`